src/NuGet.Services.AzureSearch/VersionList/IVersionListDataClient.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using NuGetGallery;
+
+namespace NuGet.Services.AzureSearch
+{
+    public interface IVersionListDataClient
```

Comment:
Do we have any documentation on the version list resource anywhere? Considering this is an "internal" resource with no public documentation, it'd be nice to have a quick explanation somewhere.

Reply:
Good point. It's in the OneNote but I'll add code comments